ios/LinkDemo-Swift/LinkDemo-Swift/ViewController.swift

```diff
@@ -48,6 +48,7 @@ class ViewController: UIViewController {
     @IBAction func didTapButton(_ sender: Any?) {
 #if USE_CUSTOM_CONFIG
         presentPlaidLinkWithCustomConfiguration()
+        // presentPlaidLinkWithCopyCustomization()
```

Comment:
No, this way people can easily try out instance configuration based customization.